### DISCUSSION TITLE
A css snippet to make the left panel and toc display a horizontal scrollbar:

### DISCUSSION THREAD

**Discussion by SiriusXT** [TriliumNext/Trilium#4706] [Show and tell]
Left panel:
```
.tree-wrapper.component .tree{
    overflow-x: auto;
    position: relative;
}
.tree-wrapper.component #ft-id-1{
    position: absolute;
}

.tree-wrapper.component .tree-actions{
    bottom: 12px;
    right: 12px;
}

::-webkit-scrollbar {
    height: 12px;
}
```
![图片](https://github.com/zadam/trilium/assets/37627919/f0934b2c-9567-414e-9bbd-4fe67b4ca550)

Toc:
```
#right-pane div.card-body div.toc-widget span.toc li{
    overflow: visible;
}
```
![图片](https://github.com/zadam/trilium/assets/37627919/de38c1e1-2208-4fbb-8db2-90a4863f0342)

Tip: Hold down the shift key and the scroll wheel to quickly scroll horizontally

**Comment by Nriver** [comment]
This CSS works great! The shift key trick is quite useful, thank you!

I believe this CSS snippet significantly enhances the mobile experience, particularly when dealing with multiple levels of note trees and lengthy note titles. It greatly simplifies note tree navigation, making it much more intuitive and user-friendly.

**Comment by tuxgirl** [comment]
Sorry for the stupid question, but... How do I turn this on?

**Reply by eliandoran** [reply]
See https://docs.triliumnotes.org/User%20Guide/User%20Guide/Theme%20development/Custom%20app-wide%20CSS.html.

**Reply by ricolandia** [reply]
> Sorry for the stupid question, but... How do I turn this on?

I'm still learning Trilium, but here is one way to get it done:
Create a note (change its type to Code > CSS) and set the label to #appCss.

**Comment by ricolandia** [comment]
Nice addition! I've incorporated into my custom css. Works great!
